terraform/shared/shared.tf.json.template.py

```diff
@@ -508,9 +508,14 @@ def conformance_pack(name: str) -> str:
                     }
                 }
                 for name, pattern in [
-                    ('clamscan', '"clamscan succeeded"'),
-                    ('freshclam', '"freshclam succeeded"'),
-                    # Using '?' to create an "a OR b" filter pattern
+                    # Using '?' to create an "a OR b" filter pattern. The filter
+                    # '?"systemd:…' is used to prevent miss success alarms
```

Comment:
I do not understand the meaning of "prevent miss success alarms". Do you mean a false positive?

Reply:
Not quite. I intended to capture the cause of the alarm (the associated metric_filter resource is unable to find an event matching a 'success' log in the most recent 24 hours, and the alarm using this filter triggers an alarm) rather than the "effect".